src/modules/GroupChannel/components/MessageInputWrapper/useHandleUploadFiles.tsx

```diff
@@ -114,23 +114,24 @@ export const useHandleUploadFiles = (
         return otherFiles.reduce(
           (previousPromise: Promise<MultipleFilesMessage | FileMessage | void>, item: File) => {
             return previousPromise.then(() => {
-              return sendFileMessage({ file: item, parentMessageId: quoteMessage?.messageId });
+              return sendFileMessage({ file: item, parentMessageId: quoteMessage?.messageId, ...params });
             });
           },
           (() => {
             if (imageFiles.length === 0) {
               return Promise.resolve();
             } else if (imageFiles.length === 1) {
-              return sendFileMessage({ file: imageFiles[0] });
+              return sendFileMessage({ file: imageFiles[0], ...params });
             } else {
-              return sendMultipleFilesMessage({
+              return sendMultipleFilesMessage?.({
                 fileInfoList: imageFiles.map((file) => ({
                   file,
                   fileName: file.name,
                   fileSize: file.size,
                   mimeType: file.type,
                 })),
                 parentMessageId: quoteMessage?.messageId,
+                ...params,
```

Comment:
Let's move `...params` to the most front.

Reply:
addressed in https://github.com/sendbird/sendbird-uikit-react/pull/1218/commits/474a8e55880595639d5b51f3fa6eb1340e2bd8f8